### DISCUSSION TITLE
Publish Pynecone web app on own domain

### DISCUSSION THREAD

**Discussion by NanoscaleSimulations** [reflex-dev/reflex#590] [Q&A]
Hi everyone

I am very new to web development and Pynecone, so please bare with me.
I am building a web app in which i wish to use some third-party python libraries. So far, i can build the app using pynecone and the third-party libraries and I also have bought a domain at one.com. However, i do not understand how i can "publish" (Dont know if it is the proper term) my pynecone web-app on my one.com domain? And how would it have access to the thrid-party python libraries that i use for my app?

Kind regards

**Comment by Lendemor** [comment]
Hello,
to deploy your website you will need :
- A server to host your website (you will install pynecone and its dependencies on this server, along with any third-party lib that you need)
- Configure your domain name to point to the IP of the server where you are doing the hosting.

Note that you will probably have to configure some kind of reverse proxy on the server hosting your site (like Nginx or Apache)

Another option is to wait for the deploy option integrated in pynecone to be ready, which will save you a lot of complexity, but it's not ready yet I think.

**Comment by NanoscaleSimulations** [comment]
> Hello,
> to deploy your website you will need :
> - A server to host your website (you will install pynecone and its dependencies on this server, along with any third-party lib that you need)
> - Configure your domain name to point to the IP of the server where you are doing the hosting.
> 
> Note that you will probably have to configure some kind of reverse proxy on the server hosting your site (like Nginx or Apache)
> 
> Another option is to wait for the deploy option integrated in pynecone to be ready, which will save you a lot of complexity, but it's not ready yet I think.

Thank you so much! At least now i know it is possible to execute my idea!
